src/background.ts

```diff
@@ -41,24 +41,32 @@ type SingleInstanceLockData = {
   filePath: string | undefined;
 };
 
+const isDevelopment = process.env.NODE_ENV !== "production";
+
+// Electronの設定ファイルの保存場所を変更
+const fixedUserDataDir = path.join(
+  app.getPath("appData"),
+  `voicevox${isDevelopment ? "-dev" : ""}`
+);
+if (!fs.existsSync(fixedUserDataDir)) {
+  fs.mkdirSync(fixedUserDataDir);
+}
+app.setPath("userData", fixedUserDataDir);
```

Comment:
`fixedUserDataDir`が下1000行のどこかで意図せず使われるかもなので、blockにして使えなくするのとかどうでしょう。
```suggestion
{
  const fixedUserDataDir = path.join(
    app.getPath("appData"),
    `voicevox${isDevelopment ? "-dev" : ""}`
  );
  if (!fs.existsSync(fixedUserDataDir)) {
    fs.mkdirSync(fixedUserDataDir);
  }
  app.setPath("userData", fixedUserDataDir);
  }
}
```
あるいは関数化するのも良いかもです。
（そのままでも問題ないです！）